handbook/people-ops/expenses.md

```diff
@@ -28,9 +28,20 @@ We use Expensify for expense report submission and approval. You will receive an
 - Add a description of your expense in the designated field. 
 
 
-### Expense categories
+### When submitting an expense
 
-Please ensure that each expense is properly categorized. Some frequently asked questions about categories include:
+- Please ensure that each expense is properly categorized. Some frequently asked questions about categories include:
+  - Software purchases or subscriptions should be categorized as "Software and SaaS <$1000"
+  - Computer purchases and other hardware (monitor, headphones, etc.) should be categorized as "Computer & Related <$1000" 
```

Comment:
We should just rename this haha... Can you email OE and ask them to remove the "<$1000" piece? They can do it for us.

Reply:
We had a thread on this a while back and there was a reason we kept it, but I just emailed them again to request the change. I'll update it once it's reflected in Expensify.